src/_includes/components/logo-orange.njk

```diff
@@ -0,0 +1,21 @@
+<svg version="1.1" id="Layer_2_00000114044064486736330470000009482575430579936388_"
```

Comment:
It makes no sense to create 2 logo.njk files for an SVG, you should be able to change the color of your SVG via CSS.

Are you still experiencing color-related issues with the SVG logo?

Reply:
Nope I merged the shapes again and now the logo changes color correctly :) 
The reason why I created 2 different logos are the following:
- when you create a separate SVG file and then give it a fill attribute you can't alter it with CSS later (it needs to be inline SVG to do so)
- the 2 logos are also different because the logo-purple one is the one I use on the navbar to go to the home and the logo-orange is used only in the mobile version and it is just decorative, not clickable. Do u think this is weird/confusing?
<img width="315" alt="Screenshot 2022-08-19 at 16 25 41" src="https://user-images.githubusercontent.com/67103707/185640201-35220980-4b0f-45a4-b54e-c7c9c84c8c8a.png">

Reply:
> when you create a separate SVG file and then give it a fill attribute you can't alter it with CSS later (it needs to be inline SVG to do so)
The solution for this is to replace your hex color code in the SVG fill with `"currentColor"`
And then go to CSS and add a color to the SVG as follows `svg {color: $purple} `
The curentColor fill will only work if there's a color assigned to it. 

I didn't realize that a couple of days ago when we where first trying to solve the SVG color issue.
But if you need some extra reading: [https://css-tricks.com/cascading-svg-fill-color/](https://css-tricks.com/cascading-svg-fill-color/)

> the 2 logos are also different because the logo-purple one is the one I use on the navbar to go to the home and the logo-orange is used only in the mobile version and it is just decorative, not clickable. Do u think this is weird/confusing?

The wrapping anchor is not a part of the logo component if I saw it correctly in your code, so it feels redundant to have 2 logo components just for the color change. 

But if you want to hardcode the color on the logo SVG, you can also do it with one component instead of 2. By turning the color into a variable that's passed or selected based on where/when you will be using the logo. 😁 
If you want to explore this option, lemme know so I can show you an example of how it's done.

Reply:
Ok I turned the chevrons back into compnents again and even with no fill attribute in the svg is working just fine! That's nice 🥳
Regarding the logo: let aside the color there are a few differences in the title... do you think these can also be included in the solution you have in mind ? Then I'd really like to see the example 🤓
<img width="1077" alt="Screenshot 2022-08-19 at 19 24 17" src="https://user-images.githubusercontent.com/67103707/185674931-87bbcff7-88ad-461f-9a68-7878f5a23d2a.png">